examples/graphql/src/boot/dbConnection.ts

```diff
@@ -1,7 +1,7 @@
 import mongoose from 'mongoose';
 
 module.exports = async app => {
-	mongoose.Promise = global.Promise;
+	(<any>mongoose).Promise = global.Promise;
```

Comment:
According to [this page](https://masteringjs.io/tutorials/mongoose/promise) (linked from the official Mongoose docs), this line isn't required anymore:
> If you see `mongoose.Promise = global.Promise` in code that uses Mongoose 5, please delete it. Mongoose 5 uses native promises by default, so that code does nothing in Mongoose 5.

Can we confirm that is the case and if so, remove this line?

Reply:
I got compilation error  `error TS2540: Cannot assign to 'Promise' because it is a read-only property.` and fixed is it such a way

Reply:
Can confirm we can remove it